devices/Hx711/Hx711.cs

```diff
@@ -14,7 +14,14 @@ namespace Iot.Device.Hx711
     public class Scale
     {
         // pulse train required to read a sample and setup gain factor for next reading
-        private readonly byte[] _readSamplePulseTrain;
+        private byte[] _readSamplePulseTrain;
+
+        // sample buffer to hold data read from DOUT
+        private byte[] _readSampleBuffer;
```

Comment:
This one should be readonly as well

Reply:
the readonly isn't showing... maybe you still need to push your changes.. 🤔